core/test/unit/services/url/utils_spec.js

```diff
@@ -438,105 +438,133 @@ describe('Url', function () {
-        });
-
-        it('api: url has subdir', function () {
-            configUtils.set({
-                url: 'http://my-ghost-blog.com/blog'
-            });
-
-            urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}, true).should.eql('http://my-ghost-blog.com/blog/ghost/api/v0.1/');
-        });
-
-        it('api: relative path is correct', function () {
-            urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}).should.eql('/ghost/api/v0.1/');
-        });
-
-        it('api: relative path with subdir is correct', function () {
-            configUtils.set({
-                url: 'http://my-ghost-blog.com/blog'
-            });
-
-            urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}).should.eql('/blog/ghost/api/v0.1/');
-        });
-
-        it('api: should return http if config.url is http', function () {
-            configUtils.set({
-                url: 'http://my-ghost-blog.com'
-            });
-
-            urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}, true).should.eql('http://my-ghost-blog.com/ghost/api/v0.1/');
-        });
-
-        it('api: should return https if config.url is https', function () {
-            configUtils.set({
-                url: 'https://my-ghost-blog.com'
-            });
-
-            urlService.utils.urlFor('api', {version: 'deprecated', versionType: 'content'}, true).should.eql('https://my-ghost-blog.com/ghost/api/v0.1/');
-        });
-
-        it('api: with cors, blog url is http: should return no protocol', function () {
-            configUtils.set({
-                url: 'http://my-ghost-blog.com'
-            });
-
-            urlService.utils.urlFor('api', {cors: true, version: 'deprecated', versionType: 'content'}, true).should.eql('//my-ghost-blog.com/ghost/api/v0.1/');
-        });
-
-        it('api: with cors, admin url is http: cors should return no protocol', function () {
-            configUtils.set({
-                url: 'http://my-ghost-blog.com',
-                admin: {
-                    url: 'http://admin.ghost.example'
-                }
+        ['deprecated', 'active'].forEach((apiVersion) => {
+            describe(`for api version: ${apiVersion}`, function () {
+                it('api: should return admin url is set', function () {
+                    configUtils.set({
+                        url: 'http://my-ghost-blog.com',
+                        admin: {
+                            url: 'https://something.de'
+                        }
+                    });
+
+                    urlService.utils
+                        .urlFor('api', {version: apiVersion, versionType: 'content'}, true)
+                        .should.eql(`https://something.de${testUtils.API.getApiPath({version: apiVersion, versionType: 'content'})}`);
```

Comment:
Are we already breaking long code lines like these to multi-lines? I saw lot of tests still have long single line code, but in these cases I broke them into multi-line as they seemed to be too long, not sure what convention we are following :)